aio/src/index.html

```diff
@@ -1,11 +1,11 @@
 <!doctype html>
-<html lang="en">
+<html lang="es">
 
 <head>
   <meta charset="utf-8">
   <title>Angular</title>
-  <meta name="Description" content="Angular is a platform for building mobile and desktop web applications.
-    Join the community of millions of developers who build compelling user interfaces with Angular.">
+  <meta name="Description" content="Angular es una plataforma para crear aplicaciones web móviles y de escritorio.
+     Únase a la comunidad de millones de desarrolladores que crean interfaces de usuario atractivas con Angular.">
```

Comment:
I feel we should use the "tu" form instead of "usted". "tu" form sounds more friendly IMO. Thoughts?

Reply:
¡Conforme!

Reply:
Actualizado, por favor revise para asegurarse de que lo hice bien.